engine/core/IgeTexture.ts

```diff
@@ -53,14 +52,6 @@ export class IgeTexture extends IgeAsset {
 		super();
 		this._loaded = false;
 
-		if (isServer) {
```

Comment:
This is definitely still required. This is a safety check for people running multiplayer games with server-side code.

Reply:
No worries, have re-worked this to keep the checks.